rust/src/protocol_types/certificates/drep_deregistration.rs

```diff
@@ -0,0 +1,43 @@
+use crate::*;
+
+#[derive(
+    Clone,
+    Debug,
+    Hash,
+    Eq,
+    Ord,
+    PartialEq,
+    PartialOrd,
+    serde::Serialize,
+    serde::Deserialize,
+    JsonSchema,
+)]
+#[wasm_bindgen]
+pub struct DrepDeregistration {
```

Comment:
Request to rename `DrepDeregistration` to `DRepDeregistration`, to be consistent with `struct DRep`.
Same for `DrepRegistration` and `DrepUpdate`.

Reply:
Thanks @itsfarseen for pointing it out, we will consider this proposal

Reply:
Renamed